contributing-guides/maintainers-guide.md

```diff
@@ -0,0 +1,37 @@
+# Maintainer's guide
+
+The following guidelines are meant to provide a general basis
+for the behavior expected of tldr-pages maintainers:
+
+- When responding to issues or pull requests,
+  remember that you're temporarily the face of the tldr-pages project.
+  **Be welcoming and friendly**, and if you don't know how to answer,
+  ping other maintainers who you think might have a say.
+
+- Although push access allows committing directly to the repository,
+  plase **create pull requests for all of your changes**,
+  except the simplest ones (e.g. typo fixes).
+  This ensures that the entire process other contributors go through
+  is exposed to maintainers, who can then identify and address bottlenecks or inconveniences.
+  Similarly, **avoid merging your own PRs**.
+
+- Ideally, **every new discussion should receive a response within 24 hours**.
```

Comment:
This is quite tight. I've been working to ~48hrs up until now. Do I need to change this?

Reply:
Yeah, the 24h was just a number that sounded reasonable on the surface when I wrote this. I didn't put too much thought into it because I assumed the "ideally" would suffice to make it clear that it's in no way an obligation, just a nice-to-have. But I can see how it could be interpreted as some sort of a deadline.

I'm perfectly fine with changing it to 28h, one week, or whatever you guys feel is reasonable for both the contributors and the maintainers.

Reply:
I am good with "ideally". Being an open-source project, I believe, by definition it does not have a deadline.

Reply:
@agnivade true....? Perhaps it could not be bold then? That would make it less imposing.